src/Bindings/LuaJson.cpp

```diff
@@ -134,7 +134,7 @@ static Json::Value JsonSerializeTable(cLuaState & a_LuaState)
 	{
 		if (lua_type(a_LuaState, -2) == LUA_TNUMBER)
 		{
-			int idx;
+			int idx = 0;
 			a_LuaState.GetStackValue(-2, idx);
```

Comment:
This call to GetStackValues should check the return code for an error. Then the uninitialized value would never be read.